Graphics fixes (adds `graphics_set`, removes `hr_version`)

Aiving opened the pull request:
In the Factorio 2.0 part of the prototypes, most of the properties responsible for graphics and animation were moved to a separate `graphics_set` property. This PR is intended to finalize the work of moving these properties to `graphics_set`.

Aiving committed:
replace the removed ways to specify graphics with the `graphics_set`

SophieFitz commented:
I've also removed all instances of `hr_version` as 2.0 deprecates this. High res is now the default. I'll make a script to remove them. Angel's uses custom functions for this so the [Remove-hr-version.py](https://github.com/tburrows13/factorio-2.0-mod-porting-guide/blob/master/remove_hr_version.py) file from the [2.0 mod porting guide](https://github.com/tburrows13/factorio-2.0-mod-porting-guide) won't work here. Unless you've already done this? (in a soon-to-be-made PR)

Aiving committed:
removed low res graphics

Aiving committed:
removed `*-hr.png` filenames

Aiving commented:
> I've also removed all instances of `hr_version` as 2.0 deprecates this. High res is now the default. I'll make a script to remove them. Angel's uses custom functions for this so the [Remove-hr-version.py](https://github.com/tburrows13/factorio-2.0-mod-porting-guide/blob/master/remove_hr_version.py) file from the [2.0 mod porting guide](https://github.com/tburrows13/factorio-2.0-mod-porting-guide) won't work here. Unless you've already done this? (in a soon-to-be-made PR)

I've decided that removing `hr_version` is generally appropriate for this PR, so now it's also dealing with that issue. I hope this doesn't hurt (?) you too much (I felt a bit like I was devaluing someone else's work)
About `remove-hr-version.py`: I agree, it is not suitable for all mods. I use another script written by me. It analyzes the lua code directly (unlike `remove-hr-version.py`, which just breaks the code into separate lines and looks for `hr_version` in them) and modifies it.

SophieFitz commented:
Fair, well it works and that's cool! 
> About remove-hr-version.py: I agree, it is not suitable for all mods. I use another script written by me. It analyzes the lua code directly (unlike remove-hr-version.py, which just breaks the code into separate lines and looks for hr_version in them) and modifies it.

Honestly, a little bit but it's due to lack of communication. Let's try do better for future stuff! Maybe what Qatavin said on Bob's, make an issue for a thing you're working on/plan to imminently work on (and I would do the same). 
> I hope this doesn't hurt (?) you too much (I felt a bit like I was devaluing someone else's work)

SophieFitz commented:
@Aiving May I ask how you did this? I would like to learn how to do it myself!
> I use another script written by me. It analyzes the lua code directly

Aiving commented:
> @Aiving May I ask how you did this? I would like to learn how to do it myself!
> 
> > I use another script written by me. It analyzes the lua code directly

Well, this is a Rust program that uses a "lossless" lua parser called `full-moon`. 
It's still not a perfect way, as it just turns the text into a tree of different elements (statements, expressions, etc) and doesn't execute the code in any way, but it more than works for me. Not sure if there is something similar outside of Rust though. But I can put the code in the public domain (which I was planning to do anyway)

SophieFitz commented:
Please! That would be awesome :)
> I can put the code in the public domain (which I was planning to do anyway)

LovelySanta commented:
how are the bigger graphics generated? Are they generated from the 3D files of angels? Or just an enlargement of the existing 2D files?

Aiving commented:
> how are the bigger graphics generated? Are they generated from the 3D files of angels? Or just an enlargement of the existing 2D files?

Most (if not all) of the machines in the angels mods already had high resolution textures (which could also be switched off in-game via mod settings, though that doesn't make sense now), so I just moved the `hr_version` content "up a level" (`animation.hr_version.*` => `animation.*`)
And deleted the old lower resolution files.
